hoodie-client/src/main/java/com/uber/hoodie/io/HoodieCreateHandle.java

```diff
@@ -126,7 +134,10 @@ public WriteStatus close() {
       stat.setFileId(status.getFileId());
       String relativePath = path.toString().replace(new Path(config.getBasePath()) + "/", "");
       stat.setPath(relativePath);
-      stat.setTotalWriteBytes(FSUtils.getFileSize(fs, path));
+      if (tempPath != null) {
+        stat.setTempPath(tempPath.toString().replace(new Path(config.getBasePath()) + "/", ""));
```

Comment:
can we add a new method `stat.setPath(relativePath, tempPath)` and to make this less verbose.. if tempPath is null, then its taken care of without an explicit null check

Reply:
In order to let `stat` to compute `tempPath`, `HoodieWriteStat` would need to know `config.getBasePath()`, then shall we do something like `stat.setPath(basePath, relativePath, tempPath)`?

Reply:
`stat.setPaths(basePath, relativePath, tempPath)` seems good to me.